### DISCUSSION TITLE
Customizing the DeleteAction on Table

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#8642] [Help] [answered]
Is it possible to set the content/text etc inside the deleteAction modal? i would like to write in it as header "Delte record X" instead of only "delete model"

**Comment by leandrocfe** [comment]
```php
->modalHeading(fn ($record): string => 'Delete ' . $record->your_column)
```

**Comment by RPotkowaVegasystems** [comment]
Ohh fine, thx. Thats works.
But is it maybe possible to change also the text?

**Reply by leandrocfe** [reply]
`->modalDescription('xx')` ?

**Comment by RPotkowaVegasystems** [comment]
thx

**Reply by leandrocfe** [reply, marked as answer]
Could you change the title of the topic to 'Customizing the DeleteAction on Table' and mark this as *answered*, please? This can help other people. Thank you

**Reply by leandrocfe** [reply]
@RPotkowaVegasystems thank you :+1:
